packages/block-library/src/comments-query-loop/block.json

```diff
@@ -34,7 +34,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
A lot of the children of the comments query loop are rendered as links. Would it be worth adding `link` as one of the defaults?